crates/starknet-devnet/tests/common/mod.rs

```diff
@@ -1,3 +1,4 @@
+#![cfg(test)]
```

Comment:
What does the `!` mean?

Reply:
it is to point that it is for the entire file

Reply:
So before this annotation placed in these files, all of these code parts were compiled into the `--release` binary?

Reply:
not, but without this attribute clippy complains about using unwraps. Its a workaround suggested in rust forums